### ISSUE TITLE
Not able to choose destination on map in online cab application

### ISSUE THREAD

**Issue by A-Shahbazi** [microg/GmsCore#587]
This is the link to app: https://play.google.com/store/apps/details?id=taxi.tap30.passenger.play
And here's the relevant part of logcat:
```
I/Timeline(  827): Timeline: Activity_launch_request id:taxi.tap30.passenger.play time:226639
I/ActivityManager(  617): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity} from pid 827
W/Tapstream( 1794): Tapstream Warning: Tapstream already instantiated, it cannot be re-created.
E/Batch   ( 1794): Batch.Push : Permission com.google.android.c2dm.permission.RECEIVE is missing, please read the documentation for more info: https://batch.com/
D/GmsMapFragImpl( 1794): onInflate
D/GmsMapFragImpl( 1794): onCreate
D/GmsMapFragImpl( 1794): onCreateView
D/dalvikvm( 1794): Trying to load lib /data/app-lib/com.google.android.gms-1/libvtm-jni.so 0x41b98c68
D/dalvikvm( 1794): Shared lib '/data/app-lib/com.google.android.gms-1/libvtm-jni.so' already loaded in same CL 0x41b98c68
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
D/GmsMapFragImpl( 1794): onTransact [unknown]: 15, android.os.Parcel@41ba2718, 0
D/GmsMapFragImpl( 1794): onResume
W/ActivityManager(  617): Unable to start service Intent { act=com.google.android.gms.measurement.START pkg=com.google.android.gms } U=0: not found
W/ActivityManager(  617): Unbind failed: could not find connection for android.os.BinderProxy@41cde030
E/GmsClient( 1794): unable to connect to service: com.google.android.gms.measurement.START on com.google.android.gms
W/GmsMapImpl( 1794): Traffic not yet supported
D/GmsMapImpl( 1794): onTransact [unknown]: 91, android.os.Parcel@41b598d0, 0
D/GmsMapImpl( 1794): getUiSettings: org.microg.gms.maps.UiSettingsImpl@41c2bc50
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
D/GmsMapImpl( 1794): unimplemented Method: setCameraMoveListener
D/GmsMapImpl( 1794): unimplemented Method: setCameraMoveStartedListener
D/GmsMapImpl( 1794): unimplemented Method: setCameraIdleListener
D/GmsMapImpl( 1794): setOnMarkerClickListener
E/RecyclerView( 1794): No adapter attached; skipping layout
W/FA      ( 1794): Service connection failed: ConnectionResult{statusCode=API_UNAVAILABLE, resolution=null, message=null}
I/ActivityManager(  617): Displayed taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity: +827ms
I/Timeline(  617): Timeline: Activity_windows_visible id: ActivityRecord{41d2d5c0 u0 taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity t4} time:227571
D/SafeParcel(  784): Unknown field num 10 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
D/GmsLocManagerSvc(  784): bound by: GetServiceRequest{serviceId=LOCATION_MANAGER, gmsVersion=10240000, packageName='taxi.tap30.passenger.play', extras=Bundle[{client_name=locationServices}]}
D/GmsMapCamUpdateFactory( 1794): newLatLngZoom
I/Timeline( 1794): Timeline: Activity_idle id: android.os.BinderProxy@426880a8 time:227942
I/System.out( 1794): 12 12 12 12
D/OnMapConfirm( 1794): applyInitialState() called with: state = [1] enabledShortcuts = [taxi.tap30.passenger.ui.widget.DestinationShortcutView{42a4ab40 V.E..... ......I. 0,0-0,0 #7f090040 app:id/button_confirmlocation_custom}]
D/OnMapConfirm( 1794): switchToConfirmOrigin() called from null
W/GmsMapImpl( 1794): MyLocation not yet supported
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
D/GmsLocManagerSvcImpl(  784): getLastLocationWithPackage: taxi.tap30.passenger.play
D/GmsLocProviderReal(  784): uh-ok: last location for network is null!
D/GmsLocProviderReal(  784): uh-ok: last location for gps is null!
D/OnMapConfirm( 1794): switchToConfirmOrigin() called from SHORTCUTS
W/PropertyValuesHolder( 1794): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder( 1794): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder( 1794): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder( 1794): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m0 with title null @ lat/lng: (xx.68876266479492,xx.389217376708984)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m1 with title null @ lat/lng: (xx.68876266479492,xx.389217376708984)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m2 with title null @ lat/lng: (xx.688621520996094,xx.39094924926758)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m3 with title null @ lat/lng: (xx.688621520996094,xx.39094924926758)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m4 with title null @ lat/lng: (xx.691184997558594,xx.388397216796875)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m5 with title null @ lat/lng: (xx.691184997558594,xx.388397216796875)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m6 with title null @ lat/lng: (xx.68806838989258,xx.385704040527344)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m7 with title null @ lat/lng: (xx.68806838989258,xx.385704040527344)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m8 with title null @ lat/lng: (xx.688873291015625,xx.39006423950195)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m9 with title null @ lat/lng: (xx.688873291015625,xx.39006423950195)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m10 with title null @ lat/lng: (xx.68994140625,xx.390811920166016)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m11 with title null @ lat/lng: (xx.68994140625,xx.390811920166016)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m12 with title null @ lat/lng: (xx.68855285644531,xx.383304595947266)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m13 with title null @ lat/lng: (xx.68855285644531,xx.383304595947266)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m14 with title null @ lat/lng: (xx.68512725830078,xx.382591247558594)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m15 with title null @ lat/lng: (xx.68512725830078,xx.382591247558594)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m16 with title null @ lat/lng: (xx.684165954589844,xx.394378662109375)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m17 with title null @ lat/lng: (xx.684165954589844,xx.394378662109375)
D/SafeParcel( 1794): Unknown field num 15 in com.google.android.gms.maps.model.MarkerOptions, skipping.
D/GmsMapMarkerImpl( 1794): New marker m18 with title null @ lat/lng: (xx.68920135498047,xx.388999938964844)
D/GmsMapCamUpdateFactory( 1794): newLatLng
W/GmsMapImpl( 1794): Compass not yet supported
W/GmsMapImpl( 1794): ZoomControls not yet supported
D/OnMapConfirm( 1794): switchToShortcuts() called in state CONFIRM_ORIGIN
```

**Comment by A-Shahbazi** [comment]
@mar-v-in 
I can still experience this using microG `0.2.6.13280`

**Comment by swiftgeek** [comment]
Happens in pl.allegro (from play store) when opening urls with it from external apps (and makes it inoperable untill hard restart of app)

```
01-03 21:03:41.458  4336  4700 D SafeParcel: Unknown field num 10 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
01-03 21:03:41.458  4336  4700 D SafeParcel: Unknown field num 11 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
01-03 21:03:41.458  4336  4700 D SafeParcel: Unknown field num 12 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
01-03 21:03:41.459  4336  4700 D GmsMeasureBrokerSvc: bound by: GetServiceRequest{serviceId=MEASUREMENT, gmsVersion=12451000, packageName='pl.allegro', extras=Bundle[{}]}
```

**Comment by A-Shahbazi** [comment]
here's the relevant part of logcat with microG `0.2.6.13280` until choosing departure point:
```
V/NlpLocationBackendFuser(  831): location=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m20s461ms {Bundle[{noGPSLocation=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m20s461ms], AVERAGED_OF=1, SERVICE_BACKEND_PROVIDER=DejaVu, SERVICE_BACKEND_COMPONENT=org.fitchfamily.android.dejavu/.BackendService, OTHER_BACKEND_RESULTS=[Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms {Bundle[{noGPSLocation=Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms], SERVICE_BACKEND_PROVIDER=ichnaea, SERVICE_BACKEND_COMPONENT=org.microg.nlp.backend.ichnaea/.BackendService}]}]]}]}]
I/Timeline(23453): Timeline: Activity_launch_request id:taxi.tap30.passenger.play time:208770608
I/ActivityManager(  608): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity} from pid 23453
W/Tapstream(22822): Tapstream Warning: Tapstream already instantiated, it cannot be re-created.
D/Batch   (22822): Batch.Push: Registration id: APA91bE5oBYNypsMe2OaHkKIID0I0L5Exivryq9FOk_9oYSJR2gYmv6XR44-JEpJjTbto2wNvVdMhnCXVeJIj8JHF8ZeZXVNaJccO3h4CL9vX2dTFZE5xXyz4H6VCkMQfCKnSk3rkWwS
D/GmsMapFragImpl(22822): onInflate
D/GmsMapFragImpl(22822): onCreate
D/GmsMapFragImpl(22822): onCreateView
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
D/GmsMapFragImpl(22822): onTransact [unknown]: 15, android.os.Parcel@4249e718, 0
D/GmsMapFragImpl(22822): onResume
W/GmsMapImpl(22822): Traffic not yet supported
D/GmsMapImpl(22822): onTransact [unknown]: 91, android.os.Parcel@41ad6178, 0
D/GmsMapImpl(22822): getUiSettings: org.microg.gms.maps.UiSettingsImpl@41a6c5e8
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
D/SafeParcel(  831): Unknown field num 10 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
D/GmsMeasureBrokerSvc(  831): bound by: GetServiceRequest{serviceId=MEASUREMENT, gmsVersion=10240000, packageName='taxi.tap30.passenger.play', extras=Bundle[{}]}
D/GmsMapImpl(22822): unimplemented Method: setCameraMoveListener
D/GmsMapImpl(22822): unimplemented Method: setCameraMoveStartedListener
D/GmsMapImpl(22822): unimplemented Method: setCameraIdleListener
D/GmsMapImpl(22822): setOnMarkerClickListener
E/RecyclerView(22822): No adapter attached; skipping layout
D/GmsMeasureSvcImpl(  831): onTransact [unknown]: 10, android.os.Parcel@41b84808, 16
I/ActivityManager(  608): Displayed taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity: +978ms
I/Timeline(  608): Timeline: Activity_windows_visible id: ActivityRecord{425b19e0 u0 taxi.tap30.passenger.play/taxi.tap30.passenger.ui.RootActivity t89} time:208771697
D/SafeParcel(  831): Unknown field num 10 in com.google.android.gms.common.internal.GetServiceRequest, skipping.
D/GmsLocManagerSvc(  831): bound by: GetServiceRequest{serviceId=LOCATION_MANAGER, gmsVersion=10240000, packageName='taxi.tap30.passenger.play', extras=Bundle[{client_name=locationServices}]}
D/GmsMapCamUpdateFactory(22822): newLatLngZoom
V/NlpLocationBackendFuser(  831): location=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m25s423ms {Bundle[{noGPSLocation=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m25s423ms], AVERAGED_OF=1, SERVICE_BACKEND_PROVIDER=DejaVu, SERVICE_BACKEND_COMPONENT=org.fitchfamily.android.dejavu/.BackendService, OTHER_BACKEND_RESULTS=[Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms {Bundle[{noGPSLocation=Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms], SERVICE_BACKEND_PROVIDER=ichnaea, SERVICE_BACKEND_COMPONENT=org.microg.nlp.backend.ichnaea/.BackendService}]}]]}]}]
W/SQLiteConnectionPool(  831): A SQLiteConnection object for database '/data/data/com.google.android.gms/databases/gcmstatus' was leaked!  Please fix your application to end transactions in progress properly and to close the database when it is no longer needed.
I/System.out(22822): 12 12 12 12
D/OnMapConfirm(22822): applyInitialState() called with: state = [1] enabledShortcuts = [taxi.tap30.passenger.ui.widget.DestinationShortcutView{4273de50 V.E..... ......I. 0,0-0,0 #7f090040 app:id/button_confirmlocation_custom}]
D/OnMapConfirm(22822): switchToConfirmOrigin() called from null
W/GmsMapImpl(22822): MyLocation not yet supported
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
D/GmsLocManagerSvcImpl(  831): getLastLocationWithPackage: taxi.tap30.passenger.play
D/OnMapConfirm(22822): switchToConfirmOrigin() called from SHORTCUTS
W/PropertyValuesHolder(22822): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder(22822): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder(22822): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
W/PropertyValuesHolder(22822): Method set() with type float not found on target class class me.zhanghai.android.materialprogressbar.IndeterminateHorizontalProgressDrawable$RectTransformX
D/GmsMapMarkerImpl(22822): New marker m0 with title null @ lat/lng: (xx.64033126831055,xx.69456481933594)
D/GmsMapMarkerImpl(22822): New marker m1 with title null @ lat/lng: (xx.64033126831055,xx.69456481933594)
D/GmsMapMarkerImpl(22822): New marker m2 with title null @ lat/lng: (xx.64079666137695,xx.69546127319336)
D/GmsMapMarkerImpl(22822): New marker m3 with title null @ lat/lng: (xx.64079666137695,xx.69546127319336)
D/GmsMapMarkerImpl(22822): New marker m4 with title null @ lat/lng: (xx.64076614379883,xx.69560623168945)
D/GmsMapMarkerImpl(22822): New marker m5 with title null @ lat/lng: (xx.64076614379883,xx.69560623168945)
D/GmsMapMarkerImpl(22822): New marker m6 with title null @ lat/lng: (xx.642791748046875,xx.695701599121094)
D/GmsMapMarkerImpl(22822): New marker m7 with title null @ lat/lng: (xx.642791748046875,xx.695701599121094)
D/GmsMapMarkerImpl(22822): New marker m8 with title null @ lat/lng: (xx.641231536865234,xx.6915168762207)
D/GmsMapMarkerImpl(22822): New marker m9 with title null @ lat/lng: (xx.641231536865234,xx.6915168762207)
D/GmsMapMarkerImpl(22822): New marker m10 with title null @ lat/lng: (xx.64291000366211,xx.69227981567383)
D/GmsMapMarkerImpl(22822): New marker m11 with title null @ lat/lng: (xx.64291000366211,xx.69227981567383)
D/GmsMapMarkerImpl(22822): New marker m12 with title null @ lat/lng: (xx.63623809814453,xx.693756103515625)
D/GmsMapMarkerImpl(22822): New marker m13 with title null @ lat/lng: (xx.63623809814453,xx.693756103515625)
D/GmsMapMarkerImpl(22822): New marker m14 with title null @ lat/lng: (xx.636192321777344,xx.693084716796875)
D/GmsMapMarkerImpl(22822): New marker m15 with title null @ lat/lng: (xx.636192321777344,xx.693084716796875)
D/GmsMapMarkerImpl(22822): New marker m16 with title null @ lat/lng: (xx.64364242553711,xx.68907165527344)
D/GmsMapMarkerImpl(22822): New marker m17 with title null @ lat/lng: (xx.64364242553711,xx.68907165527344)
V/NlpLocationBackendFuser(  831): location=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m30s451ms {Bundle[{noGPSLocation=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m30s451ms], AVERAGED_OF=1, SERVICE_BACKEND_PROVIDER=DejaVu, SERVICE_BACKEND_COMPONENT=org.fitchfamily.android.dejavu/.BackendService, OTHER_BACKEND_RESULTS=[Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms {Bundle[{noGPSLocation=Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms], SERVICE_BACKEND_PROVIDER=ichnaea, SERVICE_BACKEND_COMPONENT=org.microg.nlp.backend.ichnaea/.BackendService}]}]]}]}]
```
And here's the logcat after choosing the departure point(the app becomes unresponsive at this point):
```
D/GmsMapMarkerImpl(22822): New marker m18 with title null @ lat/lng: (xx.64113235473633,xx.694454193115234)
D/GmsMapCamUpdateFactory(22822): newLatLng
W/GmsMapImpl(22822): Compass not yet supported
W/GmsMapImpl(22822): ZoomControls not yet supported
D/OnMapConfirm(22822): switchToShortcuts() called in state CONFIRM_ORIGIN
V/NlpLocationBackendFuser(  831): location=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m35s438ms {Bundle[{noGPSLocation=Location[network xx.641131,xx.694453 acc=306 et=+2d21h9m35s438ms], AVERAGED_OF=1, SERVICE_BACKEND_PROVIDER=DejaVu, SERVICE_BACKEND_COMPONENT=org.fitchfamily.android.dejavu/.BackendService, OTHER_BACKEND_RESULTS=[Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms {Bundle[{noGPSLocation=Location[network xx.640341,xx.692180 acc=3762 et=+2d21h9m8s63ms], SERVICE_BACKEND_PROVIDER=ichnaea, SERVICE_BACKEND_COMPONENT=org.microg.nlp.backend.ichnaea/.BackendService}]}]]}]}]
```

**Comment by sm4rk0** [comment]
Duplicate of #287?

**Comment by A-Shahbazi** [comment]
> Duplicate of #287?

If I understand correctly these aren't the same because the proposed workaround(moving map so that the pointer is on the desired location) doesn't work. I'm not able to move the map at all.

**Comment by sm4rk0** [comment]
But the cause and the effect are the same. You can't pick a place on a map by tapping it.

**Comment by A-Shahbazi** [comment]
> But the cause and the effect are the same. You can't pick a place on a map by tapping it.

Yes, I can't pick a place on map but I'm not sure the cause is the same because if it is the workaround should work here too(which doesn't). Then I suspect there might be something more.

**Comment by A-Shahbazi** [comment]
This is fixed in the latest `0.2.7-17455-mapbox` version but it can be experienced in normal vtm version.
